schedule/yast/btrfs/btrfs+warnings.yaml

```diff
@@ -59,3 +62,8 @@ conditional_schedule:
         - installation/grub_test
       pvm_hmc:
         - installation/grub_test
+test_data:
+  disks:
+    - name: vda
+  error:
```

Comment:
I guess or 'errors' or just one line 'error_no_root' if it is not thought to have a list of them.

Reply:
Changed to `errors`, as this hash will contain data for another errors, that I'll introduce in the separate PRs.